frontend/lib/src/components/elements/ArrowTable/ArrowTable.tsx

```diff
@@ -42,7 +42,7 @@ export function ArrowTable(props: TableProps): ReactElement {
   return (
     <StyledTableContainer data-testid="stTable">
       {cssStyles && <style>{cssStyles}</style>}
-      <StyledTable id={cssId}>
+      <StyledTable id={cssId} data-testid="stStyledTable">
```

Comment:
Nit: Can you change these to follow our new proposed conventions? 😅

Reply:
Yes! Totally reasonable! Will do that :D